module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,24 @@
+#include <algorithm>
+#include <forward_list>
+#include <string>
+
+#include "sort.hpp"
```

Comment:
Include order should like this: file's header(sort.hpp in this case), standard library includes and other headers from project. So sort.hpp should be in the first paragraph.